sigma.css

```diff
@@ -200,7 +201,7 @@ sup {
 	padding-bottom: 22px; /* FOR MENU */
 	background: url('https://scpwiki.github.io/sigma/images/header-logo.svg')
 		10px 40px no-repeat;
-	background-size: 100px;
+	background-size: 100px 100px;
```

Comment:
I do not anticipate this causing any real issues. It's essentially adding what modern browsers already do without the extra code. So this should be fine.